File: sub-model/src/main/java/org/github/base/entity/QrtzSimpropTriggersEntity.java
package org.github.base.entity;

import java.math.BigDecimal;
import com.baomidou.mybatisplus.annotation.TableName;
import org.github.base.AbstractEntity;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.*;

/**
 * <p>
 * 
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-21
 */
@Setter
@Getter
@EqualsAndHashCode(callSuper = true)
@TableName("qrtz_simprop_triggers")
public class QrtzSimpropTriggersEntity extends AbstractEntity {
    /** UID */
    private static final long serialVersionUID = 1L;

    @TableId("sched_name")
    private String schedName;

    @TableField("trigger_name")
    private String triggerName;

    @TableField("trigger_group")
    private String triggerGroup;

    @TableField("str_prop_1")
    private String strProp1;

    @TableField("str_prop_2")
    private String strProp2;

    @TableField("str_prop_3")
    private String strProp3;

    @TableField("int_prop_1")
    private Integer intProp1;

    @TableField("int_prop_2")
    private Integer intProp2;

    @TableField("long_prop_1")
    private Long longProp1;

    @TableField("long_prop_2")
    private Long longProp2;

    @TableField("dec_prop_1")
    private BigDecimal decProp1;

    @TableField("dec_prop_2")
    private BigDecimal decProp2;

    @TableField("bool_prop_1")
    private String boolProp1;

    @TableField("bool_prop_2")
    private String boolProp2;
}

File: sub-model/src/main/java/org/github/base/entity/QrtzSimpropTriggersEntity.kt
package org.github.base.entity

import java.math.BigDecimal
import com.baomidou.mybatisplus.annotation.TableName
import org.github.base.AbstractEntity

/**
 * <p>
 * 
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-19
 */
@TableName("qrtz_simprop_triggers")
class QrtzSimpropTriggersEntity: AbstractEntity() {

    var schedName: String? = null

    var triggerName: String? = null

    var triggerGroup: String? = null

    var strProp1: String? = null

    var strProp2: String? = null

    var strProp3: String? = null

    var intProp1: Int? = null

    var intProp2: Int? = null

    var longProp1: Long? = null

    var longProp2: Long? = null

    var decProp1: BigDecimal? = null

    var decProp2: BigDecimal? = null

    var boolProp1: String? = null

    var boolProp2: String? = null

}
